uijson/import_to_las.ui.json

```diff
@@ -0,0 +1,50 @@
+{
+    "title": "Import drillhole group",
+    "geoh5": "",
+    "run_command": "las_geoh5.main",
+    "run_command_boolean": {
+        "value": false,
+        "label": "Run python module ",
+        "tooltip": "Warning: launches process to run python model on save",
+        "main": true
+    },

```

Comment:
**question:** does this need to be in ui.json? I thought it was added on the fly in the dialog by Analyst

Reply:
I think we should leave this for another issue